okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -254,27 +252,24 @@ private String tenantHeader(ProxyContext pc) {
     String tok = ctx.request().getHeader(XOkapiHeaders.TOKEN);
 
     if (auth != null) {
-      Pattern pattern = Pattern.compile("Bearer\\s+(.+)"); // Grab anything after 'Bearer' and whitespace
-      Matcher matcher = pattern.matcher(auth);
-      if (matcher.find() && matcher.groupCount() > 0) {
-        auth = matcher.group(1);
+      if (auth.startsWith("Bearer")) {
```

Comment:
https://tools.ietf.org/html/rfc6750#section-2.1 requires a single space after Bearer.
Can you check for `startsWith("Bearer ")`?

Reply:
We don't want to match BearerExtension foo.

Reply:
sure